languages/elixir/concepts/structs/about.md

```diff
@@ -4,7 +4,7 @@
 - A struct is named after the module it is defined in.
 - To define a struct use the [`defstruct`][docs-defstruct] construct.
   - The construct usually immediately follows after the module definition.
-- `defstruct` accepts either a list of [atoms][atom] (for nil default values) or key-value tuples (for specified default values).
```

Comment:
The example below says `[:engine, wings: 2]` which, to people that have no idea about keyword lists, won't look like a tuple at all. I updated the dependencies for structs to depend on keyword lists. It doesn't create any conflicts in the tree as far as I can see.